macros/SVGRef.ejs

```diff
@@ -66,7 +66,7 @@ var resultAPI = [];
             <li><%- template("domxref", [resultAPI[aTitle]]) %></li>
   <%    } 
     } %>
-   <li><a href="<%-s_svg_href%>"><%-s_svg_ref_title%></a><ol>
+   <li><details><summary><%-s_svg_ref_title%></summary><ol>
```

Comment:
The ``{{SVGRef}}`` sidebar is converted correctly 👍, as seen on https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/height.

<img width="561" alt="pr 789 svgref" src="https://user-images.githubusercontent.com/286017/45388214-f3f32c00-b5dd-11e8-9ec0-f856b54d1e0a.png">

It's collapsed by default, and is a little more interesting when expanded. But just a little.

<img width="557" alt="pr 789 svgref expanded" src="https://user-images.githubusercontent.com/286017/45388224-fbb2d080-b5dd-11e8-80ab-d952660f838b.png">